features/old/services/multiservice.feature

```diff
@@ -27,6 +27,18 @@ Feature: Multiservice feature
      And I press "Create Product"
     Then I should see "Less fancy API"
 
+  @javascript
+  Scenario: Create new product: Fail scenario
```

Comment:
We're 1 cuke short:
- Create new product: Fails with empty name
- Create new product: Fails with wrong system name

Reply:
Not really, we are asserting the flash message being shown, not the kind of error per se.

Reply:
Ok, then the test should not refer to any particular string. Meaning it should be more generic like:
```
I enter a wrong value
I click on submit
I see a flash error
```
My point is that we're asserting the errors together (I'm also considering the snapshot here) and that is wrong: there should be one test case for each field. If it's not in the specs (because of use of snapshots) then this is the right place, I think.

Reply:
I have to disagree, we are asserting the *correct* flash message was shown here, as flash messages work (first encountered, first shown) and what is most important, what the user will see.  This is a feature, not an integration or unit test... you address (almost) real user experience.
The flash messaging system is already tested, the I18n error messages as well, etc...
Then, in JS, we are testing the invalid fields and their messages with the template.

Reply:
OK